src/Geocoder/Provider/GoogleMaps.php

```diff
@@ -133,6 +138,11 @@ public function getName()
         return 'google_maps';
     }
 
+    /**
+     * @param $region
```

Comment:
This doc block is not part of the PR. You can keep this but try to avoid "code cleanups" in the same PR as features.